src/marqo/client.py

```diff
@@ -195,7 +195,14 @@ def health(self):
         mq_logger.warning('The `client.health()` API has been deprecated and will be removed in '
                           'Marqo 2.0.0. Use `client.index(index_name).health()` instead. '
                           'Check `https://docs.marqo.ai/latest/API-Reference/indexes/` for more details.')
-        return self.http.get(path="health")
+        try:
+            return self.http.get(path="health")
+        except (MarqoWebError, RequestException, TypeError, KeyError) as e:
+            raise errors.BadRequestError("Marqo encountered an error trying to check the health of the Marqo instance. "
+                                         "It is likely you are trying to check the health on Marqo Cloud V2. "
+                                         "The `client.health()` API is not supported on Marqo Cloud V2 and will be removed in"
+                                         "Marqo 2.0.0. Please Use `client.index(index_name).health()` instead. "
```

Comment:
you be careful about spaces. This will become `inMarqo 2.0.0`

Also, for the example, let's make it clearer (as index_name is a local variable, users might get confused):
`client.index("my-index").health()`